internal/config/config.go

```diff
@@ -15,18 +14,22 @@ const (
 )
 
 var c *Config
+var myContainerID string
+
+func init() {
+	myContainerID = "1111111"
```

Comment:
just curious, why do you need something not "" ?

Reply:
I'll change it, no reason